apis/r/R/SOMADataFrame.R

```diff
@@ -38,11 +38,21 @@ SOMADataFrame <- R6::R6Class(
       for (field_name in index_column_names) {
         field <- schema$GetFieldByName(field_name)
 
+        # TODO: Parameterize
+        tile_extent <- 2048L
```

Comment:
recent tiledb-soma-py bug: if index type is int8/uint8 this mods to 0 😬 -- maybe use 64 in that case

Reply:
Good call, thanks!